src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts

```diff
@@ -478,4 +480,49 @@ describe('CoursePlayerComponent', () => {
     component.contentProgressEvent(telemetryEvent);
     expect(courseConsumptionService.updateContentsState).not.toHaveBeenCalled();
   });
+  it('should show join training popup if course is unenrolled and try to play content', () => {
```

Comment:
Move the duplicate code in both specs to a method and call that method

Reply:
In both specs, there is different code